### PR TITLE
before delete Udf check exist

### PR DESCRIPTION
**Pull request by samz406**
## What is the purpose of the pull request

1 Modify judgment order(调整顺序，先判断是否存在)
2 before delete check entity exist

### CODE REVIEW

File: dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/UdfFuncService.java [hunk 1]
```diff
@@ -303,7 +303,12 @@ private boolean checkUdfFuncNameExists(String name){
     @Transactional(rollbackFor = Exception.class)
     public Result delete(int id) {
         Result result = new Result();
-
+        //check exist
```
Comment (Technoboy-) [inline dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/UdfFuncService.java]: What is the benefit of the checking logic before deleting ?

Comment (samz406) [reply dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/UdfFuncService.java]: I have expressed a problem, this is not to say that the delete order is adjusted. The delete method modifies the question. Order adjustment refers to the checkUdfFuncNameExists method. My understanding is that you should first determine if udf exists and then check if hdfs is started.
我表述的有问题，这里不是说delete顺序调整。delete方法修改的是提示问题。顺序调整是指checkUdfFuncNameExists方法。我的理解是应该先判断udf是否存在然后再去检查hdfs是否启动。

Comment (Technoboy-) [reply dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/UdfFuncService.java]: For delete method only, in line 304.
What is the benefit of the checking logic before deleting ?

Comment (Technoboy-) [reply dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/UdfFuncService.java]: all the checking logic is nice above line 304 .

**Pull request merged**